src/main/java/org/reaktivity/nukleus/http2/internal/Http2Stream.java

```diff
@@ -191,9 +190,16 @@ private void doReset(
         connection.closeStream(this);
     }
 
-    void sendHttp2Window(int update)
+    void sendHttpWindow()
     {
-        targetWindow -= update;
+        long maxWindow = Math.min(http2OutWindow, connection.factory.bufferPool.slotCapacity());
+        // target already has stream.httpOutWindow, calculate how much more it can send
+        long more = maxWindow - httpOutWindow;
```

Comment:
Suggest renaming `more` to `windowDelta`

Reply:
done